currency_exchange/app.py

```diff
@@ -2,47 +2,42 @@
 import argparse
 import requests
 
-
 CURRENCIES = {
```

Comment:
we no longer need this dict.

After you remove it, we can no longer limit choices in args.currency as we used to:
```
    parser.add_argument('currency', type=str,
                        choices=CURRENCIES.keys(), help='currency type')
```

Please read https://realpython.com/python-exceptions/ and handle the case when a user requests a currency that doesn't exist so `r = requests.get(f'http://api.nbp.pl/api/exchangerates/rates/A/{currency}/')` fails

Reply:
Done